src/ios/CDVSplashScreen.m

```diff
@@ -60,6 +60,63 @@ - (void)observeValueForKeyPath:(NSString*)keyPath ofObject:(id)object change:(NS
     [self updateImage];
 }
 
+- (UIColor *)colorFromHexString:(NSString *)hexString {
```

Comment:
Does Cordova not export any methods to handle color parsing in preferences? (I don't know -- have't looked). If it does, I'd suggest using those, rather than building your own. If not, a few thoughts:

1. Would it make sense to support the case where the user forgets "#"?
2. Would it make sense to support the backgroundColor pref case where it uses "0x" instead?

Reply:
For the moment i will not change this part because i don't know what is the better solution.

Reply:
statusbar plugin uses a similar code for this. It's in the plugin, not provided by Cordova. It also checks that the color is a string (it's not a problem here as it's a preference, so I think we can be sure that it's a string. But also checks the length and the #

Reply:
Statusbar handles all of this in js here: https://github.com/apache/cordova-plugin-statusbar/blob/master/www/statusbar.js#L75
including shortcut css, like #345 => #334455